active/NavigationView/BottomItems/BottomMenuItems.md

```diff
@@ -0,0 +1,155 @@
+
+# Background
+Today, [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) provides a place in its [Pane](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) to show any number of items.
```

Comment:
Do we have an Anatomy of a NavigationView picture somewhere? (Picture of the control with labels pointing out the parts.)

Reply:
I haven't seen any out there, but @YuliKl would know for sure. I can create one if needed.

Reply:
[This](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/images/leftnav-anatomy.png) look appropriate?

Reply:
Sorry, I should have known that one was out there. I can add that to this section for some context.